netcore/System.Private.CoreLib/src/System/Reflection/CustomAttribute.cs

```diff
@@ -46,11 +44,7 @@ static class CustomAttribute
 		static bool IsUserCattrProvider (object obj)
 		{
 			Type type = obj as Type;
-#if !FULL_AOT_RUNTIME
 			if ((type is RuntimeType) || (RuntimeFeature.IsDynamicCodeSupported && type is TypeBuilder))
-#else
```

Comment:
This will keep TypeBuilder alive so it cannot be linked out.

Reply:
I guess we'll need to teach the linker about this then, or is there another option?

Reply:
you can do `tb = corelib.GetType ("TypeBuilder")` and then check that `tb.IsAssignableFrom (type)`

Reply:
> corelib.GetType ("TypeBuilder")
that would break linker, please leave it for now as-is